display_order.c

```diff
@@ -0,0 +1,77 @@
+//
+// Created by Irina on 07-Nov-19.
+//
+
+#include <stdio.h>
+#define NR_DRINKS 5
+#define NR_FOODS 3
```

Comment:
why are these defined here? if you don't want them to define them in main, just create a constants.h and define all your constants there...

Reply:
I created a constants.h file. I hope it is OK now.